adminSiteServer/apiRouter.ts

```diff
@@ -3138,17 +3138,6 @@ getRouteNonIdempotentWithRWTransaction(
 postRouteWithRWTransaction(apiRouter, "/images", async (req, res, trx) => {
     const { filename, type, content } = validateImagePayload(req.body)
 
-    const preexisting = await trx<DbEnrichedImage>("images")
```

Comment:
This is a change that Daniel suggested - it's annoying to be told to rename an image, only to find out that the hash collides after.

So now we check for the has collision first.